src/fender_value.rs

```diff
@@ -324,6 +340,24 @@ impl ToString for FenderValue {
                     .collect::<Vec<_>>()
                     .join(", ")
             ),
+            FenderValue::Struct(s) => {
+                format!(
+                    "{}{{{}}}",
```

Comment:
???

Reply:
what?

Reply:
what is `"{}{{{}}}"` doing

Reply:
the name of the struct followed by `{}` with the fields passed in

Reply:
`{{` and `}}` are how you escape curly braces in format strings